src/bindings/system/fs.rs

```diff
@@ -88,6 +88,14 @@ pub fn init(lua: &Lua) -> ::Result<()> {
         create_symlink(src_path, symlink_dest).map_err(LuaError::external)
     })?)?;
 
+    module.set("remove_recursive", lua.create_function( |_, path: String| {
+        fs::remove_dir_all(&path).map_err(LuaError::external)
+    })?)?;
+
+    module.set("touch", lua.create_function( |_, path: String| {
+        fs::write(&path, "").map_err(LuaError::external)
```

Comment:
While this would work to create a file, the problem would be is that internally, rust calls `File::create` which would create a file if it doesnt exist or replace it if it does (pretty much truncating it). `touch` only would create a file if it doesnt exist and do nothing if it does. You could use `OpenOptions` to make sure it is created if it doesnt exist and do nothing if it does. Eg `OpenOptions::new().write(true).create(true).open(&path).map(|_| ()).map_err(LuaError::external)` would create the file without truncating the file to 0.